documentation/guides/migrating-from-v3-to-v4.md

```diff
@@ -0,0 +1,390 @@
+# Migrating from v3 to v4
+
+Polaris v4 contains mostly internal changes and has simple migration paths. This guide contains multiple steps on how to deal with our breaking changes in your application.
+
+**Note:** This migration guide is for v3 but may apply for v2 as well.
+
+- [Testing](#testing)
+- [API Changes](#api-changes)
+  - [AppProvider](#appprovider)
+    - [i18n](#i18n)
+  - [Navigation](#navigation)
```

Comment:
I'd recommend prefixing all of these anchors because for example `#navigation` already exists in the style guide.

Reply:
Afaik we can't prefix them since they're based on the headers. I'm not 100% positive though, would love to hear how, if there is one 😄

Reply:
Yeah there is! Named anchors should do the trick